docs/docusaurus/docs/cloud/quickstarts/airflow_quickstart.md

```diff
@@ -0,0 +1,114 @@
+description: Connect GX Cloud to an Airflow Orchestrator.
+---
+
+In this quickstart, you'll learn how to use GX Cloud with Apache Airflow. Apache Airflow is an orchestration tool that allows you to schedule and monitor your data pipelines. For more information about Apache Airflow, see the [Apache Airflow documentation](https://airflow.apache.org/docs/apache-airflow/stable/index.html).
+
+## Prerequisites
+
+- You have a [GX Cloud Beta account](https://greatexpectations.io/cloud).
+
+- You have the [Astro CLI](https://docs.astronomer.io/astro/cli/overview) installed.
+
+- You have [connected GX Cloud to a Data Asset on a Data Source](/docs/cloud/data_assets/manage_data_assets#create-a-data-asset).
+
+- You have [created an Expectation Suite](/docs/cloud/expectation_suites/manage_expectation_suites.md) and [added Expectations](/docs/cloud/expectations/manage_expectations#create-an-expectation).
+
+- You have [added a Checkpoint to your Expectation](/docs/cloud/checkpoints/manage_checkpoints#add-a-checkpoint).
+
+
+## Create a local Airflow project and set dependencies
+
+1. Open a terminal, navigate to the directory where you want to create your Airflow project, and then run the following code:
+
+    ```bash title="Terminal input"
+    mkdir gx-cloud-airflow && cd gx-cloud-airflow
+    astro dev init
+    ```
+    After running the code, a new directory is created, you're taken to that directory, and a new Airflow project is initialized.
+
+2. Browse to the directory where you created your Airflow project, open the `requirements.txt` file, and then add the following text as a new line: 
+
+    ```
+    airflow-provider-great-expectations==0.2.7
+    ```
+
+    This text adds the GX Airflow Provider to the Airflow project.
+    
+3. Save your changes and close the `requirements.txt` file.
+
+4. Open the `packages.txt` file and add the following text as a new line:
+
+    ```
+    libgeos-c1v5
+    ```
+    This text adds the `libgeos-c1v5` library to the Airflow project.
+
+5. Save your changes and close the `packages.txt` file.
+
+## Create a DAG file for your GX Cloud Checkpoint
+
+1. Open a terminal, browse to the `dags` folder of your Airflow project, and then run the following code to create a new DAG named `gx_dag.py`:
+
+    ```bash title="Terminal input"
+    touch gx_dag.py
+    ```
+
+2. Open the `gx_dag.py` DAG file and add the following code:
+
+    ```python 
+    import os
+    import great_expectations as gx
+    from airflow import DAG
+    from airflow.operators.python import PythonOperator
+    from datetime import datetime
+
+    def run_gx_airflow():
+        os.environ["GX_CLOUD_ACCESS_TOKEN"] = "<YOUR_ACCESS_TOKEN>"
+        os.environ["GX_CLOUD_ORGANIZATION_ID"] = "<YOUR_CLOUD_ORGANIZATION_ID>"
+
+    # Replace YOUR_ACCESS_TOKEN and YOUR_CLOUD_ORGANIZATION_ID with your values. See [Get your user access token and organization ID](/docs/cloud/set_up_gx_cloud#get-your-user-access-token-and-organization-id).
```

Comment:
Just a stylistic suggestion - I'd recommend:
* Moving this comment above where the constants are defined (so under the `def run_gx_airflow()` line)
* Breaking the single line comment into multiple lines (example below), as the super long comment is rendered on one line in the Docusaurus code box and requires awkward side scroll

```
# Replace YOUR_ACCESS_TOKEN and YOUR_CLOUD_ORGANIZATION_ID with your
# GX Cloud credentials. See this docs page for more details:
# https://docs.greatexpectations.io/docs/cloud/set_up_gx_cloud#get-your-user-access-token-and-organization-id
```

Also: the markdown link format doesn't render in the code block, and it's currently linking to a local URL.

Reply:
@rachhouse typically, we add statements about replacing parameters _after_ the code sample. 

With regard to your concern about the super long comment, when I reviewed the content this morning in a browser, the comment rendered correctly and auto-sized as expected. I've broken the text, but I don't think it's necessary.

I've updated the link.